devices/SPS30/SHDLC/SHDLCProtocol.cs

```diff
@@ -0,0 +1,177 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Ports;
+
+namespace Iot.Device.SPS30.SHDLC
+{
+    /// <summary>
+    /// Minimal implementation of the SHDLC protocol used by Sensirion. This implementation is based upon the information
```

Comment:
minor comment: for comments, best to finish the sentences on comments with a dot.